src/lib_ccx/ccx_common_constants.h

```diff
@@ -117,42 +117,42 @@ enum ccx_stream_type
 	CCX_STREAM_TYPE_ISO_IEC_13818_6_TYPE_B = 0x0B,
 	CCX_STREAM_TYPE_ISO_IEC_13818_6_TYPE_C = 0x0C,
 	CCX_STREAM_TYPE_ISO_IEC_13818_6_TYPE_D = 0x0D,
-	CCX_STREAM_TYPE_AUDIO_AAC   = 0x0f,
-	CCX_STREAM_TYPE_VIDEO_MPEG4 = 0x10,
-	CCX_STREAM_TYPE_VIDEO_H264  = 0x1b,
-	CCX_STREAM_TYPE_PRIVATE_USER_MPEG2=0x80,
+	CCX_STREAM_TYPE_AUDIO_AAC   = 0x0f, // Should there really be 0x0f?
+	CCX_STREAM_TYPE_VIDEO_MPEG4 = 0x10, // Should there really be 0x10?
+	CCX_STREAM_TYPE_VIDEO_H264  = 0x1b, // Should there really be 0x1b?
```

Comment:
All these numbers are just MPEG constants, they come from MPEG standards. 
https://exiftool.org/TagNames/M2TS.html

Reply:
Ok. I'll add a comment on these being reserved MPEG constants.
Just not so clear or well-known constants (like 404 error, for example), hence it may be little confusing to see [0...n-1] pattern and then "random" numbers.